jakarta/transform.sh

```diff
@@ -34,7 +34,7 @@ if [ "${REWRITE_OFFLINE-false}" != "true" ]; then
   # Build Kotlin Maven Plugin to allow skipping main compilation
   # (skipping test compilation is supported but not main)
   rm -rf target/kotlin
-  git clone -b v1.6.21-jakarta --depth 1 https://github.com/gsmet/kotlin.git target/kotlin
+  git clone -b v1.7.0-jakarta --depth 1 https://github.com/gsmet/kotlin.git target/kotlin
```

Comment:
@gsmet I suppose you'll have to do something to make this work?

Reply:
Yes, I will have to. I need to create a PR for the Kotlin plugin. Maybe they will accept it.